.env.local

```diff
@@ -100,4 +100,4 @@ ITW_IDP_HINT_TEST=YES
 # IPZS Privacy Policy URL
 ITW_IPZS_PRIVACY_URL='https://io.italia.it/informativa-ipzs'
 # ITW Documents on IO URL
-ITW_DOCUMENTS_ON_IO_URL='https://io.italia.it/documenti-su-io'
+ITW_DOCUMENTS_ON_IO_URL='https://io.italia.it/documenti-su-io'
```

Comment:
I think we don't need this env. Wrong merge?

Reply:
Yes, but it's strange. Thanks for noticing